Bottom: 14ef07c0bc9ac3220a9d106f91f2417eb3985117
Top:    5a93c640aed5fd906089e010160e33c7e6c40649
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 16:00:53 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
